docs/output.md

```diff
@@ -12,30 +12,121 @@ The directories listed below will be created in the results directory after the
+Metapep uses a relational data model that consists of tables that can describe relationships between different objects.
 
-![MultiQC - FastQC sequence counts plot](images/mqc_fastqc_counts.png)
+### Download proteins
 
-![MultiQC - FastQC mean quality scores plot](images/mqc_fastqc_quality.png)
+<details markdown="1">
+<summary>Output files</summary>
+
+* `entrez_data/`
+    * `entities_proteins.entrez.tsv`: matches temporary protein id given by Entrez to entities. Contains protein_tmp_id and entity_name.
+    * `microbiomes_entities.entrez.tsv`: matches entities (taxa) and their weights to microbiomes. Contains microbiome_id, entity_id and entity_weight for unique microbiome - entity combinations downloaded from Entrez.
+    * `proteins.entrez.tsv.gz`: contains protein_tmp_id (protein id given by Entrez) and protein_sequence for all proteins downloaded from Entrez.
+    * `taxa_assemblies.tsv`: matches taxon id to assembly id.
+
+</details>
+
+Proteins are downloaded for input type taxa from Entrez.
+
+### Prodigal
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `prodigal/`
+    * `*.gff`: contains proteins predicted by Prodigal in gff format.
+    * `proteins.pred_*.tsv.gz`: contains proteins predicted by Prodigal in tsv format. The columns are protein_tmp_id (<contig-id_suffix>) and protein_sequence.
+
+</details>
+
+Proteins are predicted for input type assembly and bins.
+
+### Generate peptides
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `db_tables/`
+    * `peptides.tsv.gz`: contains peptide_id and peptide_sequence for all unique peptides.
+
+</details>
+
+Peptides are generated for downloaded or predicted proteins.
 
-![MultiQC - FastQC adapter content plot](images/mqc_fastqc_adapter.png)
+### Report stats
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `db_tables/`
+    * `stats.txt`: contains statistics: unique protein counts, total peptide counts, unique peptide counts, unique peptides across all conditions.
+
+</details>
+
+Some statistics on protein and peptide number are calculated.
+
+### Epitope prediction
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `db_tables/`
+    * `predictions.tsv.gz`: contains peptide_id, prediction_score (epitope prediction score) and allele_id for all unique peptide - allele combinations.
+* `logs/`
+    * `prediction_warnings.log`: contains warnings that occured during epitope prediction.
+
+</details>
+
+Epitopes are predicted for unique peptide - allele combinations.
+
+### Plot results
+
+<details markdown="1">
+<summary>Output files</summary>
+
+* `figures/`
+    * `entity_binding_ratios/`
+        * `entity_binding_ratios.allele_*.tsv`: Contains condition_name, binding_rate and entity_weight. The binding rate is calculated per entity as number of binders divided by total number of peptides. Multiple occurences of peptides within one protein are not counted.
```

Comment:
What stands `*` for? Allele mentioned in the sample sheet? Would be good to mention that, if possible. If I am right, it could be:
```suggestion
        * `entity_binding_ratios.allele_*.tsv`: Plots the entity binding ratios per allele. Contains condition_name, binding_rate and entity_weight. The binding rate is calculated per entity as number of binders divided by total number of peptides. Multiple occurences of peptides within one protein are not counted.
```

Reply:
Yes, it is the specific allele, makes sense to put it like this.